packages/rehype-plugin-codeblock/src/helpers.ts

```diff
@@ -0,0 +1,38 @@
+import { Comment, DocType, Element, Text } from 'hast-format';
```

Comment:
Sorry seems like we need to activate license checking on CI. But, meanwhile, can we have it here? Thank you!